app/(studio)/success-page/page.tsx

```diff
@@ -0,0 +1,5 @@
+import SuccessPage from '@/components/contact/SuccessPage';
+
+const Page = () => <SuccessPage />;
+
+export default Page;
```

Comment:
You could update "Page" to be more descriptive and reflect a SuccessPage.

Reply:
I changed it to Success, as SuccessPage is imported at the top